src/config/dynamic-theme-fixes.config

```diff
@@ -3261,6 +3261,10 @@ a.external::after
 ul.main-menu .top-level-entry::before
 
 CSS
+.copy-icon {
+    background: var(--darkreader-neutral-text) url(/static/media/clippy.92fffda9.svg) 50% no-repeat !important;
```

Comment:
You can remove this line and add 2 new new sections:

```CSS
INVERT
.copy-icon

CSS
[...]

IGNORE IMAGE ANALYSIS
.main-page-content .code-example .copy-icon
```

Reply:
For some reasons, I didn’t aware of **IGNORE IMAGE ANALYSIS** rule. It seems to work with your suggestion.